### PR TITLE
Application Timeline - Experience step

### LINKED ISSUE #4061: Applicant - Build full UI of My Experience step of the new Application
This needs to assemble the experience form modals and experience accordions into one page, with save and continue buttons. It should be preview-able in storybook for now.

# Images
https://xd.adobe.com/view/4ac43ee7-a919-4397-6888-ce98019603f7-3cf5/screen/5e2d2165-3b00-48aa-94e9-1f77b8dbb7fe/Step-2B-Requirements-15

### PR DESCRIPTION
**Pull request by yonikid15**
Resolves #4061 

### Notes:
- You can check it out on storybook at "My Experience". The demo allows creating and editing an education experience. As well as deleting for all. 
- There are still a couple warnings that I'm looking into. Hopefully, you'll see what I'm missing.

### CODE REVIEW

File: resources/assets/js/components/Application/Experience/Experience.tsx [hunk 1]
```diff
@@ -0,0 +1,742 @@
...
+    irrelevantSkillCount: number,
+    relevantSkills: RelevantSkill[],
+    handleEdit: () => void,
+    handleDelete: () => void,
+  ): React.ReactElement => {
+    const education = experience as ExperienceEducation;
+    const educationType =
+      educationTypes.find(({ id }) => education.education_type_id === id)?.name[
+        locale
+      ] || "Education type not found";
```
Comment (Khristophor) [inline resources/assets/js/components/Application/Experience/Experience.tsx:273]: Need to localize these error messages, as well as the general `<p>Error</p>` tag further down

File: resources/assets/js/components/Application/Experience/Experience.tsx [hunk 2]
```diff
@@ -0,0 +1,742 @@
...
+} from "../ExperienceModals/AwardExperienceModal";
+import ExperienceEducationAccordion from "../ExperienceAccordions/ExperienceEducationAccordion";
+import ExperienceWorkAccordion from "../ExperienceAccordions/ExperienceWorkAccordion";
+import ExperienceCommunityAccordion from "../ExperienceAccordions/ExperienceCommunityAccordion";
+import ExperiencePersonalAccordion from "../ExperienceAccordions/ExperiencePersonalAccordion";
+import ExperienceAwardAccordion from "../ExperienceAccordions/ExperienceAwardAccordion";
+import { mapToObject, hasKey } from "../../../helpers/queries";
+import { ExperienceSkill as RelevantSkill } from "../ExperienceAccordions/BaseExperienceAccordion";
+
+// TODO: Move method to Experience selectors file or utility file.
```
Comment (Khristophor) [inline resources/assets/js/components/Application/Experience/Experience.tsx:51]: I can migrate these into the helpers file I created in the Review PR if you'd like

File: resources/assets/js/components/Application/Experience/Experience.tsx [hunk 3]
```diff
@@ -0,0 +1,742 @@
...
+            startDate={personal.start_date}
+            title={personal.title}
+          />
+        );
+      case "experience_award":
+        return (
+          <ExperienceAwardAccordion
+            key={`${award.id}-${award.type}`}
+            // TODO: Add awardLink field to Award Modal Form?
+            awardLink={{ text: "My Award", title: "my award", url: "/" }}
```
Comment (Khristophor) [inline resources/assets/js/components/Application/Experience/Experience.tsx:372]: I think I had removed this field from the Award Accordion in the Review PR, seems to have been added as an oversight to the markup.

I can also fix this up in my Review PR if you'd like.

File: resources/assets/js/components/Application/Experience/Experience.tsx (hunk 2 above)
Comment (yonikid15) [reply resources/assets/js/components/Application/Experience/Experience.tsx:51]: Ya sure, thank you 👍

File: resources/assets/js/components/Application/Experience/Experience.tsx (hunk 3 above)
Comment (yonikid15) [reply resources/assets/js/components/Application/Experience/Experience.tsx:372]: Again, sounds good 👍

**Pull request merged**